common/static/common/js/components/collections/paging_collection.js

```diff
@@ -79,6 +79,9 @@
              * underlying server API.
              */
             getPage: function () {
+                // TODO: this.currentPage is currently returning a function sometimes when it is called.
```

Comment:
OK, this is the story. When this method is first called (when this code is in use, it is typically called many times), this.currentPage often returns the function defined above in paginator_ui instead of a number. However, later on in the call stack it starts returning the number. I've verified that bookmarks (which use this code) still behave appropriately. I've also verified that I see the same behavior when I set a breakpoint in production.